docs/development-tips.md

```diff
@@ -98,11 +98,23 @@ This tool exposes various useful pieces of information about things like HTTP he
 Django settings, SQL queries, and template variables. Note that running with the toolbar on
 may have an impact on local server performance.
 
-### TIP: Updating the documentation
+### TIP: Updating this documentation
 
-Our documentation is written as Markdown files and served in GitHub pages
+This documentation is written as Markdown files and served in GitHub pages
 by [mkdocs](https://www.mkdocs.org/user-guide/deploying-your-docs/).
 
 Every time a PR is merged to master, Travis will build and deploy the documentation to https://cfpb.github.io/cfgov-refresh. See [How we use Travis CI](https://github.com/cfpb/cfgov-refresh/blob/master/docs/travis.md) for more info.
 
-To add new pages to the navigation, edit the [mkdocs.yml](https://github.com/cfpb/cfgov-refresh/blob/master/mkdocs.yml) file in the root directory.
\ No newline at end of file
+To add new pages to the navigation, edit the [mkdocs.yml](https://github.com/cfpb/cfgov-refresh/blob/master/mkdocs.yml) file in the root directory.
+
+The source Markdown file used to generate one page of reference documentation
+can be generated dynamically from the source code by running the following
+Django management command from within a Python environment:
```

Comment:
The wording here is a bit awkward. Could we be state explicitly what is generated from source code with something like:

> The markdown file for the StreamField blocks documentation is generated from source code. To generate that documentation, run the following Django management command etc:

Reply:
:+1: Implemented in b1c2b28.